src/installation/live-images/partitions.md

```diff
@@ -19,9 +19,12 @@ The following sections will detail the options for partition configuration.
 ## BIOS system notes
 
 It is recommended that you create an MBR partition table if you are using a BIOS
-boot system. This will limit the number of partitions you create to four. It is
-possible to install a GPT partition table on a BIOS system, but grub will need a
-special partition to boot properly.
+boot system. This will limit the number of partitions you create to four.
+
+It is possible to use a GPT partition table on a BIOS system, but grub will
+require a special partition to boot properly. This partition should be at the
+beginning of your disk and have a size of 1MB, with type BIOS boot. Don't create
```

Comment:
Is `BIOS Boot` the name in fdisk/cfdisk? I wonder if it would be useful to provide the hex byte here for readers?

Reply:
Is it `ef` - `EFI (FAT-12/16/`?

Reply:
Sorry, I got this backwards in my head, but still I think this would be useful with a little more detail. But until there are more specific details I would rather not include this.

Reply:
`BIOS boot` is the name in cfdisk, yeah. Is the hex byte the long-ish string cfdisk shows at the bottom of the partition type selection screen?

Well I guess it wouldn't be so long if it's just a byte lol

Reply:
Check out [this post](https://askubuntu.com/questions/703443/gdisk-hex-codes)

I'm guessing you mean adding the GUID since this is GPT we're talking about, is that right?

Reply:
Yes. Sorry I don't touch EFI much when I don't need to, and never run GPT without it. 😅